From 2abb938de386c8841bcd1fdafce8b4ed2e26a67d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=B6rg=20Thalheim?= <joerg@thalheim.io>
Date: Thu, 31 Aug 2023 00:58:29 +0200
Subject: [PATCH] drop password
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Jörg Thalheim <joerg@thalheim.io>
---
 internal/cmd/root.go | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/internal/cmd/root.go b/internal/cmd/root.go
index 1af24fb..00c5eaf 100644
--- a/internal/cmd/root.go
+++ b/internal/cmd/root.go
@@ -158,14 +158,14 @@ func openPostgreSQLDatabase(ctx context.Context) (database.DB, error) {
 	host, _ := os.LookupEnv("SHIORI_PG_HOST")
 	port, _ := os.LookupEnv("SHIORI_PG_PORT")
 	user, _ := os.LookupEnv("SHIORI_PG_USER")
-	password, _ := os.LookupEnv("SHIORI_PG_PASS")
+	// password, _ := os.LookupEnv("SHIORI_PG_PASS")
 	dbName, _ := os.LookupEnv("SHIORI_PG_NAME")
 	sslmode, _ := os.LookupEnv("SHIORI_PG_SSLMODE")
 	if sslmode == "" {
 		sslmode = "disable"
 	}
 
-	connString := fmt.Sprintf("host=%s port=%s user=%s password=%s dbname=%s sslmode=%s",
-		host, port, user, password, dbName, sslmode)
+	connString := fmt.Sprintf("host=%s port=%s user=%s dbname=%s sslmode=%s",
+		host, port, user, dbName, sslmode)
 	return database.OpenPGDatabase(ctx, connString)
 }
-- 
2.41.0

